src/AtomsGaffer/AtomsCrowdGenerator.cpp

```diff
@@ -1164,9 +1164,6 @@ ConstPathMatcherDataPtr AtomsCrowdGenerator::computeBranchSet( const ScenePath &
 	{
 		agentPath.back() = agentName;
 
-        std::vector<std::string> paths;
```

Comment:
John  H: `AtomsCrowdGenerator::computeBranchSet()` appears to copy the entire contents of the set into a `paths` vector, and then do nothing but throw it away.